packages/x-components/src/design-system/utilities/position.scss

```diff
@@ -0,0 +1,15 @@
+.x-static {
+  position: static;
+}
+.x-fixed {
+  position: fixed;
+}
+.x-absolute {
+  position: absolute;
+}
```

Comment:
We can change the position property with a class, but what about the top, left, bottom, and right properties? Tailwind adds [CSS classes](https://tailwindcss.com/docs/top-right-bottom-left) for them. Should we also ad  `!important`?